pymongo/common.py

```diff
@@ -684,6 +699,7 @@ def validate_auto_encryption_opts_or_none(option: Any, value: Any) -> Optional[A
     "uuidrepresentation": validate_uuid_representation,
     "waitqueuemultiple": validate_non_negative_integer_or_none,
     "waitqueuetimeoutms": validate_timeout_or_none,
+    "datetime_conversion": validate_datetime_conversion,
```

Comment:
Let's document the datetime_conversion parameter in `MongoClient.__init__`. We should add a link to  :ref:`handling-out-of-range-datetimes` there too.